transiXNOR/plotting.py

```diff
@@ -125,42 +125,69 @@
 
 if (PLOT_CURRENT or PRINT_CURRENT_ONLY):
```

Comment:
Can you create an new procedure called PLOT_CURRENT_EXT for your new plots from -0.1 to 0.3 V and keep the original PLOT_CURRENT the same? Because for publication, we only need to plot from 0 to 0.2 V. Thanks!
Same for PLOT_FAMILY_CURVES -> PLOT_FAMILY_CURVES_EXT.